src/types/rtti.rs

```diff
@@ -262,6 +267,31 @@ impl Drop for Class {
     }
 }
 
+#[repr(transparent)]
+pub struct ClassFlags(red::CClass_Flags);
+
+impl fmt::Debug for ClassFlags {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        f.debug_struct("ClassFlags")
+            .field("abstract", &self.0.isAbstract())
+            .field("native", &self.0.isNative())
+            .field("scripted class", &self.0.isScriptedClass())
```

Comment:
```suggestion
            .field("is_scripted_class", &self.0.isScriptedClass())
```
I'd try to make the names be valid identifiers (no spaces), also I generally prefer to prefix booleans with `is_`, `has_` etc

Reply:
done in 439db76